app/javascript/vue/my_module/description.vue

```diff
@@ -23,7 +23,7 @@
           objectType="MyModule"
           :objectId="parseInt(myModule.id, 10)"
           :fieldName="'my_module[description]'"
-          :lastUpdated="myModule.attributes.updated_at"
+          :lastUpdated="myModule.attributes.updated_at_seconds"
```

Comment:
why we need it in seconds?

Reply:
This for tinymce, it used to show unsaved changes. Comparing with last draft time.